src/Orleans/Logging/LoggingUtils.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orleans.Runtime
+{
+    public class LoggingUtils
+    {
+        private const string LogCodeString = "OrleansLogCode";
+        ///In order to user message bulking feature, Application needs to use this method to create EventId, 
+        /// and also build their loggerProvider on top of OrleansLoggerDecorator 
+        public static EventId CreateEventId(int errorCode)
+        {
+            return new EventId(errorCode, LogCodeString);
+        }
+
+        internal static EventId CreateEventId(ErrorCode errorCode)
+        {
+            return new EventId((int)errorCode, LogCodeString);
```

Comment:
I would just drop passing "OrleansLogCode" to every EventId, as it adds no value. At most, I would pass `errorCode.ToString()` if we want to preserve the name (we were not doing that in the past, but it might be an OK improvement, not sure about the side effects).
In the future, we should probably just replace this ErrorCode enum with a class that has all the EventIds as static members.